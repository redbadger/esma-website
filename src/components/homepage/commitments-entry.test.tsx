import React from "react";
import renderer from "react-test-renderer";
import CommitmentsEntry from './commitments-entry';

describe("CommitmentsEntry", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <CommitmentsEntry content={<div>test</div>} image={<Icon/>} shiftRight={false} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

const Icon = () => {
  return <img />;
};
