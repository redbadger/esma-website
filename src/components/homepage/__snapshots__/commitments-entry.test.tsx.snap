// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommitmentsEntry renders correctly 1`] = `
<div
  className="css-5c99pg-commonStyles-commitmentsEntryStyles-CommitmentsEntry"
>
  <div
    className="left-shift"
  >
    <img />
    <div>
      test
    </div>
  </div>
</div>
`;
